### PR TITLE
Detector: Unchecked `send()` on address

### PR DESCRIPTION
**Pull request by TilakMaddy**

### CODE REVIEW

File: aderyn_core/src/detect/high/unchecked_send.rs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    fn severity(&self) -> IssueSeverity {
+        IssueSeverity::High
+    }
+
+    fn title(&self) -> String {
+        String::from("Unchecked `bool success` value for send call.")
+    }
+
+    fn description(&self) -> String {
+        String::from("`address(payable?).send(address)` returns a boolean that must be used to verify if the transaction was successful")
```
Comment (alexroan) [inline aderyn_core/src/detect/high/unchecked_send.rs:59]: We want to teach devs the reasons behind each detector, so the description should include:
* Why this is dangerous
* What the mitigation is

**Review by alexroan — CHANGES_REQUESTED** [review]

File: aderyn_core/src/detect/high/unchecked_send.rs (hunk 1 above)
Comment (TilakMaddy) [reply aderyn_core/src/detect/high/unchecked_send.rs:59]: Understood

Comment (TilakMaddy) [reply aderyn_core/src/detect/high/unchecked_send.rs]: ✅

Comment (alexroan) [reply aderyn_core/src/detect/high/unchecked_send.rs]: Beautiful description @TilakMaddy ! 😍

**Review by alexroan — CHANGES_REQUESTED** [review]
@TilakMaddy It looks like the reportgen is non-deterministic here.

Can you take a look?

**Comment by TilakMaddy** [comment]
but Ci has passed tho 🤔

File: aderyn_core/src/detect/high/unchecked_send.rs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    fn name(&self) -> String {
+        IssueDetectorNamePool::UncheckedSend.to_string()
+    }
+}
+
+#[cfg(test)]
+mod unchecked_send_tests {
+    use crate::detect::{detector::IssueDetector, high::unchecked_send::UncheckedSendDetector};
+
+    #[test]
```
Comment (alexroan) [inline aderyn_core/src/detect/high/unchecked_send.rs:77]: @TilakMaddy `#[serial]`!

**Pull request merged**